AudioMessageFeatures/AudioMessageFeatures/MessageViewController.swift

```diff
@@ -0,0 +1,290 @@
+//
+//  ViewController.swift
+//  AudioMessageFeatures
+//
+//  Created by Ezra Black on 5/5/20.
+//  Copyright © 2020 Casanova Studios. All rights reserved.
+//
+
+import UIKit
+import AVFoundation
+class RecordingTableViewCell: UITableViewCell {
+    
+    @IBOutlet weak var recordingNameLabel: UILabel!
+    @IBOutlet weak var timeRecordedLabel: UILabel!
+    @IBOutlet weak var recordingDurationLabel: UILabel!
+    @IBOutlet weak var playButton: UIButton!
+    @IBOutlet weak var recordButton: UIButton!
+    @IBOutlet weak var deleteButton: UIButton!
+    @IBOutlet weak var timeSlider: UISlider!
+    @IBOutlet weak var audioVisualizer: AudioVisualizer!
+    
+}
+
+class MessageViewController: UIViewController {
```

Comment:
This is a refactoring thing, so definitely secondary, but this class could probably be split up a bit into...
- a custom `AudioRecorderPlayer` class with playback/recording business logic
- a custom `UIControl` subclass that interfaces with and provides a UI for the custom player class
- this view controller contains an instance of the custom control and anything not included there

This might be a fun challenge to do at the end of the week, after the sprint challenge, or if you finish the other module project work early.